danielMunteanu/src/main/java/Address.java

```diff
@@ -0,0 +1,26 @@
+public class Address {
```

Comment:
![image](https://user-images.githubusercontent.com/60616588/78000606-335aa800-733d-11ea-862f-90daa004ba93.png)
Take a look on your commit message. is it good? I think it can be done in another way, try to think about a good message.

Reply:
I will take that in count on next commit.

Reply:
please also update PR description

Reply:
done